cmd/v2_test_objects.go

```diff
@@ -388,17 +388,19 @@ func (p *QueryTransformationPipeline) ParseQuery(message any) (*model.ExecutionP
 	// TODO this is a hack to create a table for the query
 	// Why parser needs a table?
 	tableName := "test_table"
-	table, err := clickhouse.NewTable(`CREATE TABLE `+tableName+`
-		( "message" String, "@timestamp" DateTime64(3, 'UTC'), "attributes_values" Map(String,String))
-		ENGINE = Memory`,
-		clickhouse.NewNoTimestampOnlyStringAttrCHConfig(),
-	)
-	if err != nil {
-		return nil, err
+	table := clickhouse.Table{
```

Comment:
I still like the idea of having this behind constructor, esp. given that this repeats in few places.

Of course I'd make `clickhouse.NewTable()` take table name and list of columns not the `CREATE TABLE ...` string 😉